topics/sequence-analysis/tutorials/de-novo-rad-seq/tutorial.md

```diff
@@ -177,35 +185,70 @@ Run `Stacks: De novo map` Galaxy tool. This program will run ustacks, cstacks, a
+>    sample_CCCC	1
+>    sample_CCAA	1
+>    sample_CCTT	1
+>    sample_CCGG	1
+>    sample_CACA	1
+>    sample_CAAC	1
+>    sample_CATG	1
+>    sample_CAGT	1
+>    sample_CTCT	2
+>    sample_CTAG	2
+>    sample_CTTC	2
+>    sample_CTGA	2
+>    sample_GGGG	2
+>    sample_GGAA	2
+>    sample_GGTT	2
+>    sample_GGCC	2
+>    ```
+>    {% include snippets/create_new_file.md %}
+>
+> 2. **Stacks: De novo map** {% icon tool %}: with the following parameters:
+>    - *"Select your usage"*: `Population`
+>    - *"Files containing an individual sample from a population"*: `The demultiplexed reads (collection)`
+>    - *"Specify a population map"*: `the population map you just created`
+>    - Section **Assembly options**
+>      - *"Minimum number of identical raw reads required to create a stack"*: `3`
+>      - *"Number of mismatches allowed between loci when building the catalog"*: `3`
+>      - *"Remove, or break up, highly repetitive RAD-Tags in the ustacks program"*: `Yes`
 >
-> Once Stacks has completed running, investigate the output files: `result.log` and `catalog.` (snps, alleles and tags).
+> 3. Once Stacks has completed running, investigate the output files: `result.log` and `catalog.` (snps, alleles and tags).
 >
 >    ![The output of de novo map](../../images/RAD4_Population_Genomics/denovo/denovo_out.png)
 >
+> > ### {% icon tip %} Tip: data formatting
+> >
+> > If you are using a file presenting population information and individual name in a different manner than expected by STACKS, you can use Galaxy tools like `Replace Text` (for example to replace `Rabbit Slough` by a population number like `2`, `Add column` (for example to add `sample_`) or `Cut columns from a table` (to put the new `sample_` column af the first place) and finally `Regex replace` (replacing `(sample_)\t` by `\1`) to generate it...
+>    {: .tip}
+>
+>
 {: .hands_on}
 
 # Calculate population genomics statistics
 
 > ### {% icon hands_on %} Hands-on: Calculate population genomics statistics
 >
-> **Stacks: populations** {% icon tool %}: Run the last step of **Stacks: De novo map** pipeline specifying data filtering options (minimum percentage of individuals in a population required to process a locus for that population: 0.75 , output options (VCF and Structure) and enabling SNP and haplotype-based F statistics calculation.
+> 1. **Stacks: populations** {% icon tool %} with the following parameters:
+>    - Section **Data filtering options**
+>      - *"Minimum percentage of individuals in a population required to process a locus for that population"*: `0.75`
+>    - Section **Output options** (VCF and Structure) and enabling SNP and haplotype-based F statistics calculation.
+>      - *"Output results in Variant Call Format (VCF)"*: `Yes`
+>      - *"Output results in Structure Format"*: `Yes`
+>    - *"Enable SNP and haplotype-based F statistics"*: `Yes`
 >
 >    ![The output of the populations tool](../../images/RAD4_Population_Genomics/denovo/populations_log.png)
```

Comment:
is this the correct image? the alt text implies it should be the output of the tool? but it is a screenshot of the job information menu?